config/default.json

```diff
@@ -52,6 +52,9 @@
       "disableNest": false,
       "disableGym": false,
       "disableFortUpdate": false,
+    //    invasion hook processing for Golbat systems
+      "confirmedInvasion": false,
```

Comment:
I don't understand these options from this name/description, so I doubt others will :-)

Reply:
confirmedInvasion -> if true, the second invasion hook from golbat, with lineup in it will be processed
disableNotConfirmedInvasion -> if true the first invasion hook from golbat will be NOT processed
so it is possible to use it, as it is / process both hook, first one when invasion ist found, second when lineup is scanned / only invasion hooks with scanned lineups
btw. the both hook option works fine with `Poracle replies to itself #690` PR

Reply:
i am open to suggestions for the key name

Reply:
So is this turning it on or disabling it? If turning it on, it could be "processConfirmedInvasionLineups" ?
disableUnconfirmedInvasion is probably better for the one below?
The disable tags should likely be together

Reply:
changed - test in progress